api-spec.md

```diff
@@ -1,26 +1,28 @@
-OAFeat you can return any `Feature`, which is a geometry plus any number of properties. In the STAC specification an 
-`Item` is a `Feature`, with additional required fields for `datetime` and `assets`. OAFeat also defines the concept of 
-a Collection, which contains Features. A STAC `Collection` aligns with (and extends slightly) a OAFeat `Collection`; it 
-contains Items.
-
-In OAFeat Collections are the sets of data that can be queried 
-([7.11](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#_collections_)), and each describes basic information 
-about the geospatial dataset, like its name and description, as well as the spatial and temporal extents of all the data 
-contained. [STAC collections](../collection-spec/README.md) contain this same information, along with other STAC 
-specific fields and thus are compliant with both OAFeat Collections and STAC Collections and is returned from the 
-`/collections/{collection_id}` endpoint.
+The OGC API - Features is a standard API that represents collections of geospatial data. It defines the RESTful interface 
+to query geospatial data, with GeoJSON as a main return type. With OAFeat you can return any `Feature`, which is a geometry 
+plus any number of properties. The core [STAC Item spec](https://github.com/radiantearth/stac-spec/item-spec/README.md) 
+enhances the core `Feature` with additional requirements and options to enable cataloging of spatiotemporal 'assets' like 
+satellite imagery. This STAC `Item` always links to an asset, and these assets always have a capture time, so it requires 
+fields for `datetime` and `assets`. The STAC API extends the OAFeat core with some key functionality to enable search of 
+geospatial assets, detailed below.
+
+OAFeat also defines the concept of a Collection, which contains Features. In OAFeat Collections are the sets of data that can 
+be queried ([7.11](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#_collections_)), and each describes basic 
+information about the geospatial dataset, like its name and description, as well as the spatial and temporal extents of all 
+the data contained. [STAC collections](https://github.com/radiantearth/stac-spec/collection-spec/README.md) contain this same 
+information, along with other STAC specific fields to provide additional metadata for searching spatiotemporal assets, and 
+thus are compliant with both OAFeat Collections and STAC Collections and are returned from the `/collections/{collection_id}` 
+endpoint.
 
 In OAFeat Features are the individual records within a Collection and are provided in GeoJSON format. 
-[STAC Items](../item-spec/README.md) are analogous to OAFeat Features, are in GeoJSON, and are returned from the 
+[STAC Items](https://github.com/radiantearth/stac-spec/item-spec/README.md) are analogous to OAFeat Features, are in GeoJSON, and are returned from the 
```

Comment:
@cholmes Those kind of links are all invalid... they need to include the version/tree, but I feel like that's quite annoying to update. Wondering whether we should include the stac-spec as Git submodule here? Then it behaves like a subfolder, links are simple and link to the right API version. What do you think @matthewhanson ?

Reply:
Sub-module would definitely be interesting, especially if we can tie that to a specific version. I know these ones need to change, but was thinking to just do master -> master until we do the first release, and then start tieing them to specific versions, since structures may change. So I'd lean towards using master and then just file an issue that they need to be updated to tie to a version.

Reply:
I could do the sub-module thing... It's fixed to a commit (i.e. version). All other options seem like duplicated work